pkg/sql/catalog/table_elements.go

```diff
@@ -432,8 +438,19 @@ type ConstraintToUpdate interface {
 	// there is one.
 	UniqueWithoutIndex() descpb.UniqueWithoutIndexConstraint
 
+	// PrimaryKey returns the underlying index descriptor backing the UNIQUE constraint,
+	// if there is one.
```

Comment:
nit: can you add a `TODO(postamar): return interface instead of protobuf` comment here and for `Unique()` please?

Reply:
I will just do it here myself.